<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AkCalculatorMainView</class>
 <widget class="QMainWindow" name="AkCalculatorMainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>551</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="hBoxCalculate">
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="btnCalculate">
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="notebook">
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabSummary">
       <attribute name="title">
        <string>Summary</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabGraphs">
       <attribute name="title">
        <string>Graphs</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabJournal">
       <attribute name="title">
        <string>Journal</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,4">
        <item>
         <layout class="QHBoxLayout" name="hBoxTop" stretch="0,3">
          <item>
           <layout class="QVBoxLayout" name="hBoxCalendar">
            <item>
             <widget class="QCalendarWidget" name="calendar">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="hBoxButtonsOKClear">
              <item>
               <widget class="QPushButton" name="btnOK">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>OK</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnClear">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Clear</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="textNote">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListWidget" name="listNotes"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionInstrumentView"/>
    <addaction name="actionHistoricalView"/>
    <addaction name="separator"/>
    <addaction name="actionConnectionsView"/>
    <addaction name="actionOptionsView"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuPanels">
     <property name="title">
      <string>Panels</string>
     </property>
     <addaction name="actionSummary"/>
     <addaction name="actionGraphs"/>
     <addaction name="actionJournal"/>
    </widget>
    <addaction name="menuPanels"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../resources/menuIcons.qrc">
     <normaloff>:/icons/menu/help.bmp</normaloff>:/icons/menu/help.bmp</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources/menuIcons.qrc">
     <normaloff>:/icons/menu/about.bmp</normaloff>:/icons/menu/about.bmp</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../resources/menuIcons.qrc">
     <normaloff>:/icons/menu/connect.bmp</normaloff>:/icons/menu/connect.bmp</iconset>
   </property>
   <property name="text">
    <string>Connect...</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="../resources/menuIcons.qrc">
     <normaloff>:/icons/menu/disconnect.bmp</normaloff>:/icons/menu/disconnect.bmp</iconset>
   </property>
   <property name="text">
    <string>Disconnect...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInstrumentView">
   <property name="icon">
    <iconset resource="../resources/menuIcons.qrc">
     <normaloff>:/icons/menu/instruments.bmp</normaloff>:/icons/menu/instruments.bmp</iconset>
   </property>
   <property name="text">
    <string>Instrument Manager...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionHistoricalView">
   <property name="icon">
    <iconset resource="../resources/menuIcons.qrc">
     <normaloff>:/icons/menu/history.bmp</normaloff>:/icons/menu/history.bmp</iconset>
   </property>
   <property name="text">
    <string>Historical Data Manager...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionConnectionsView">
   <property name="icon">
    <iconset resource="../resources/menuIcons.qrc">
     <normaloff>:/icons/menu/connections.bmp</normaloff>:/icons/menu/connections.bmp</iconset>
   </property>
   <property name="text">
    <string>Connection Manager...</string>
   </property>
  </action>
  <action name="actionOptionsView">
   <property name="icon">
    <iconset resource="../resources/menuIcons.qrc">
     <normaloff>:/icons/menu/options.bmp</normaloff>:/icons/menu/options.bmp</iconset>
   </property>
   <property name="text">
    <string>Options...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSummary">
   <property name="text">
    <string>Summary</string>
   </property>
  </action>
  <action name="actionGraphs">
   <property name="text">
    <string>Graphs</string>
   </property>
  </action>
  <action name="actionJournal">
   <property name="text">
    <string>Journal</string>
   </property>
  </action>
  <action name="actionJidem_na_Samui">
   <property name="text">
    <string>Jidem na Samui</string>
   </property>
  </action>
  <action name="actionNeeeee">
   <property name="text">
    <string>Neeeee</string>
   </property>
  </action>
  <action name="actionJoy">
   <property name="text">
    <string>Joy</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btnCalculate</tabstop>
  <tabstop>notebook</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/menuIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>released()</signal>
   <receiver>listNotes</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>calendar</receiver>
   <slot>showSelectedDate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>textNote</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>textNote</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
